packages/paste-core/components/button-group/src/ButtonGroup.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+import {styled, css} from '@twilio-paste/styling-library';
+
+const AttachedStyled = styled.span(
+  css({
+    '& > button:focus, & > button:hover': {
+      // NOTE: allow hover and focus indication to remain visible on all sides of a button despite
+      // sibling elements being rendered immediately adjacent under the attached styles
+      zIndex: 'zIndex10',
+    },
+    '& > button:not(:first-of-type):not(:last-of-type)': {
+      borderRadius: 0,
+    },
+    '& > button:first-of-type': {
+      borderTopRightRadius: 0,
+      borderBottomRightRadius: 0,
+    },
+    '& > button:last-of-type': {
+      borderTopLeftRadius: 0,
+      borderBottomLeftRadius: 0,
+    },
+  })
+);
+
+const UnattachedStyled = styled.span(
+  css({
+    '& > button:not(:first-of-type)': {
+      marginLeft: 'space30',
```

Comment:
suggestion(non-blocking): Change to space40 since that's what most of the button layouts are looking like in 1Twilio work: https://www.figma.com/file/OUHEaNwLPqXYg0vZ3smJDA/Side-by-side-component-work?node-id=228%3A91013

fwiw this is non-blocking because we can also get to this when we make all our 1Twilio component updates later in the year, but it also it seems like it could be straightforward enough to adjust now. Will leave it up to you.